packages/react-components/babel-preset-storybook-full-source/src/index.js

```diff
@@ -0,0 +1,11 @@
+// This is internal code and should be a dev dependency
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { registerTsProject } = require('nx/src/utils/register');
+
+// This is internal code and should be a dev dependency
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { workspaceRoot } = require('@nrwl/devkit');
+
+registerTsProject(workspaceRoot, 'tsconfig.base.json');
+
+module.exports = require('./index.ts');
```

Comment:
lets use `eslint-disable` pragma

```suggestion
// This is internal code and should be a dev dependency
/* eslint-disable import/no-extraneous-dependencies */
const { registerTsProject } = require('nx/src/utils/register');

const { workspaceRoot } = require('@nrwl/devkit');

registerTsProject(workspaceRoot, 'tsconfig.base.json');
/* eslint-enable import/no-extraneous-dependencies */

module.exports = require('./index');
```